samples/extauthz/src/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 Istio Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+# build a ext-authz binary using the golang container
+FROM golang:1.14.2 as builder
```

Comment:
Is there a reason to use an older go version that has known security fixes in later releases? Maybe those issues aren't affecting this, but perception says a later 1.14 or 1.15 release might be better.

Reply:
yeah, that makes sense, updated to 1.15.